lib/components/Editor/index.js

```diff
@@ -37,10 +38,21 @@ const Tiptap = (
 ) => {
   const [isImageUploadVisible, setImageUploadVisible] = useState(false);
 
+  let isPlaceholderActive = false;
+  if (placeholder) {
+    const placeholderKeysLength = Object.keys(placeholder).length;
+    isPlaceholderActive =
+      typeof placeholder === "string" ||
+      (placeholderKeysLength === 1
+        ? !placeholder.title
```

Comment:
A confusion here. when `typeof placeholder === "string" ` is truthy, we are trying to access `placeholder.title` (line 47). but we should not expect a string value to have a property `title` right?